src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -16,30 +17,76 @@ const adminCondition = [
 	}
 ];
 
-const modules = {
-	jiraDevelopmentTool: {
-		application: {
-			value: "GitHub"
-		},
-		capabilities: [
-			"branch",
-			"commit",
-			"pull_request"
-		],
-		actions: {
+interface JiraDevelopmentToolActions {
+	createBranch?: {
+		templateUrl: string;
+	};
+	searchConnectedWorkspaces?: {
+		templateUrl: string;
+	};
+	searchRepositories?: {
+		templateUrl: string;
+	};
+	associateRepository?: {
+		templateUrl: string;
+	};
+}
+
+const CREATE_BRANCH_ENDPOINT =
+	`${envVars.APP_URL}/create-branch-options?issueKey={issue.key}&issueSummary={issue.summary}&tenantUrl={tenant.url}&jwt={jwt}&addonkey=${envVars.APP_KEY}`;
+const SEARCH_CONNECTED_WORKSPACES_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/search`;
+const SEARCH_REPOSITORIES_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/repositories/search`;
+const ASSOCIATE_REPOSITORY_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/repositories/associate`;
+
+export const getGenericContainerUrls = async (): Promise<string[] | null> => {
+	if (await booleanFlag(BooleanFlags.ENABLE_GENERIC_CONTAINERS)) {
+		return [
+			CREATE_BRANCH_ENDPOINT,
+			SEARCH_CONNECTED_WORKSPACES_ENDPOINT,
+			SEARCH_REPOSITORIES_ENDPOINT,
+			ASSOCIATE_REPOSITORY_ENDPOINT
+		] as string[];
+	}
+
+	return null;
+};
+
+const defineJiraDevelopmentToolModuleActions = async (): Promise<JiraDevelopmentToolActions> => {
+	if (await booleanFlag(BooleanFlags.ENABLE_GENERIC_CONTAINERS)) {
+		return {
 			createBranch: {
-				templateUrl: `${envVars.APP_URL}/create-branch-options?issueKey={issue.key}&issueSummary={issue.summary}&tenantUrl={tenant.url}&jwt={jwt}&addonkey=${envVars.APP_KEY}`
+				templateUrl: CREATE_BRANCH_ENDPOINT
 			},
 			searchConnectedWorkspaces: {
-				templateUrl: `${envVars.APP_URL}/jira/workspaces/search`
+				templateUrl: SEARCH_CONNECTED_WORKSPACES_ENDPOINT
 			},
 			searchRepositories: {
-				templateUrl: `${envVars.APP_URL}/jira/workspaces/repositories/search`
+				templateUrl: SEARCH_REPOSITORIES_ENDPOINT
 			},
 			associateRepository: {
-				templateUrl: `${envVars.APP_URL}/jira/workspaces/repositories/associate`
+				templateUrl: ASSOCIATE_REPOSITORY_ENDPOINT
 			}
+		};
+	} else {
+		return {
+			createBranch: {
+				templateUrl: CREATE_BRANCH_ENDPOINT
+			}
+		};
+	}
+};
+
+const	modules = {
+	jiraDevelopmentTool: {
+		application: {
+			value: "GitHub"
 		},
+		capabilities: [
```

Comment:
on a outside this PR note, what are these used for(capabilities)?

Reply:
No idea 😓 . This isn't part of my PR. A bunch of the stuff here is marked as green because the additions I've made reformatted/moved some shit around so it is coming as a change. This was already here and otherwise untouched.